packages/apps-config/src/api/spec/index.ts

```diff
@@ -138,6 +138,7 @@ const spec: Record<string, OverrideBundleDefinition> = {
   'jupiter-prep': jupiter,
   'jupiter-rococo': jupiterRococo,
   'kilt-parachain': kilt,
+  khala: phalaParachain,
```

Comment:
Just move one line up, before kilt.... CI is complaining.

Reply:
Fixed